src/components/login/login.tsx

```diff
@@ -1,17 +1,35 @@
+import electron from "electron";
```

Comment:
Change this to directly import remote.

```suggestion
import { remote } from "electron";
```